app/models/katello/content_view_version.rb

```diff
@@ -280,6 +280,11 @@ def validate_destroyable!(skip_environment_check = false)
       true
     end
 
+    def all_hosts_editable?(lifecycle_environment)
+      ::Host.in_content_view_environment(self.content_view, lifecycle_environment).count ==
+          ::Host.authorized("view_hosts").in_content_view_environment(self.content_view, lifecycle_environment).count
```

Comment:
Weird question but what happens if there are 0 hosts. And user is not authorized mess with any hosts. Wont this still return true.